tests/comments.test.js

```diff
@@ -0,0 +1,219 @@
+const request = require("supertest");
+const initApp = require("../server.js");
+const mongoose = require("mongoose");
+const commentModel = require("../models/comments_model.js");
+const postModel = require("../models/posts_model.js");
+
+let app;
+beforeAll(async () => {
+  app = await initApp();
+});
+beforeEach(async () => {
+  await commentModel.deleteMany();
+});
+
+afterAll(async () => {
+  mongoose.connection.close();
+});
```

Comment:
when parameter is missing
when post does not exist
when post_id is missing
when there are no comments